extensions/visual-studio-code-project-manager/src/search-project-manager-projects.tsx

```diff
@@ -255,3 +269,13 @@ function isGitRepo(path: string): boolean {
   const gitConfig = config.sync({ cwd: path, path: ".git/config", expandKeys: true });
   return !!gitConfig.core;
 }
+
+function isRemotePreject(path: string): boolean {
```

Comment:
@TobiasYin Please fix typo "isRemoteProject"

Reply:
Thanks, I don't notice this. Already fixed.